src/main/java/dev/morling/onebrc/CalculateAverage_royvanrijn.java

```diff
@@ -50,30 +63,99 @@ private double round(double value) {
         }
     }
 
-    public static void main(String[] args) throws IOException {
+    public static void main(String[] args) throws Exception {
+
+        long before = System.currentTimeMillis();
+
+        // Split the file into numberOfChunks, use random access to read and process in parallel.
+
+        int numberOfChunks = 42; // Of course.
```

Comment:
Why is 42 obvious?

Reply:
It is the answer to life, the universe and everything.